conf/layer.conf

```diff
@@ -12,7 +12,7 @@ BBFILE_PRIORITY_sota = "7"
 LAYERDEPENDS_sota = "openembedded-layer"
 LAYERDEPENDS_sota += "meta-python"
 LAYERDEPENDS_sota += "filesystems-layer"
-LAYERSERIES_COMPAT_sota = "kirkstone langdale mickledore nanbield"
+LAYERSERIES_COMPAT_sota = "scarthgap"
```

Comment:
I would at least keep nanbield since this commit is also included there, and there the release is also still supported by upstream (and we don't have a dedicated branch here).

Reply:
yeah make sense to keeping nanbield for some more time